packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -653,7 +653,8 @@ export class MgtPerson extends MgtTemplatedComponent {
     if (imageSrc && !this._isInvalidImageSrc && this._avatarType === 'photo') {
       return html`
         <div class="img-wrapper">
-          <img alt=${title} src=${imageSrc} @error=${() => (this._isInvalidImageSrc = true)} />
+          <img tabindex=0 alt=${personDetails.displayName} src=${imageSrc} @error=${() =>
```

Comment:
@jobala this should be `this.personDetails.displayName` or `personDetailsInternal.displayName`. Fails the build so I'm wondering how it's not caught locally with `yarn build`.

Reply:
Thanks @musale, fixing.